packages/app-staking/src/Account/Bond.tsx

```diff
@@ -40,16 +41,17 @@ class Bond extends React.PureComponent<Props, State> {
     const { accountId, controllerId } = this.props;
 
     this.state = {
+      controllerError: null,
       controllerId: controllerId ? controllerId.toString() : accountId,
       destination: 0
     };
   }
 
   render () {
     const { accountId, isOpen, onClose, t } = this.props;
-    const { bondValue, controllerId, destination } = this.state;
+    const { bondValue, controllerError, controllerId, destination } = this.state;
     const hasValue = !!bondValue && bondValue.gtn(0);
-    const canSubmit = hasValue && !!controllerId;
+    const canSubmit = hasValue && !!controllerId && !controllerError;
```

Comment:
`controllerId` is actually always true once all components are mounted, so there's no real need for the check.